### DISCUSSION TITLE
M5Stack Core2 BOTTOM 2 led control

### DISCUSSION THREAD

**Discussion by rtemkollos** [HASwitchPlate/openHASP#394] [Hardware]
How to start using data pin 25 for LED backlight.  Is it possible to somehow tie up the FastLed library?  So far, I'm just figuring out your wonderful product.![3_6d1c790d-09ab-43bc-8894-7b4e5fc296fa_1200x1200.jpg](https://user-images.githubusercontent.com/72490425/196659318-932011bd-5b6f-4bd3-9579-a32db3af5f05.jpg)

**Comment by fvanroie** [comment]
> How to start using data pin 25 for LED backlight. Is it possible to somehow tie up the FastLed library?

There is [no support for addressable LEDs](https://github.com/HASwitchPlate/openHASP/discussions/263) in openHASP at this time.
You can add libraries and [custom code](https://www.openhasp.com/0.6.3/compiling/custom/) to the project though.
